types/twitter.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	"github.com/dghubble/go-twitter/twitter"
+)
+
+// TwitterResponse main format for data coming out of twitter api
+type TwitterResponse struct {
+	BaseMessage
```

Comment:
Not sending the BaseMessage back anymore

Reply:
Instead we are sending the configs back - so switch it with `TwitterConfig` and make sure to add it in the `Data()` method as well, otherwise looks good